site/en/blog/prerender-pages/index.md

```diff
@@ -0,0 +1,271 @@
+    document.addEventListener('prerenderingchange', resolve);
+  } else {
+    resolve();
+  }
+});
+
+async function initAnalytics() {
+  await whenActivated;
+  gtag('config', 'UA-12345678-1');
+}
+
+initAnalytics();
+```
+
+This is the method we have implemented on this site and on [https://web.dev](https://web.dev).
+
+### Measuring performance
+
+For measuring performance metrics, analytics should consider whether it is better to measure these based upon the activation time rather than the page load time that browser APIs will report.
+
+For Core Web Vitals, measured by Chrome through the [Chrome User Experience Report](/docs/crux/), these are intended to measure the user experience. So these are measured based on activation time. This will often result in a 0 second LCP for example, showing this is great way of improving your Core Web Vitals.
+
+From version 3.0.4, the [`web-vitals` library](https://github.com/GoogleChrome/web-vitals) has been updated to handle prerendered navigations in the same way Chrome measure Core Web Vitals. This version also flags prerendered navigations for those metrics in the [`Metric.navigationType`](https://github.com/GoogleChrome/web-vitals#metric) attribute, which is discussed next.
+
+### Measuring prerenders
+
+Whether a page is prerendered can be seen with a non-zero `activationStart` entry of [`PerformanceNavigationTiming`](https://developer.mozilla.org/docs/Web/API/PerformanceNavigationTiming), supplemented with changes for some navigation currently not reported by that. This can then be logged using a Custom Dimension, or similar when logging the page views:
+
+```js
+// Set Custom Dimension for Prerender status
+const prerendered =
+    performance &&
+    performance.getEntriesByType &&
+    performance.getEntriesByType('navigation')[0] &&
+    performance.getEntriesByType('navigation')[0].activationStart > 0;
+gtag('set', {'dimension1': prerendered});
+// Initialise GA - including sending page view by default
+gtag('config', 'UA-12345678-1');
+```
+
+This will allow your analytics to show how many navigation are prerendered compared to other types of navigation, and also allow you to correlation any performance metrics or business metrics to these different navigation types. Faster pages means happier users, which can often have real impact on business measures as our [case studies](https://web.dev/tags/case-study/) show.
+
+As you measure the business impact of prerendering pages for instant navigations, you can decide whether it is worth investing more effort in using this technology to allow more navigations to be prerendered, or to investigate why pages are not being prerendered.
+
+#### Measuring hit rates
+
+As well as measuring the impact of pages that are visited after a prerender, it is also important to measure pages that are prerendered and _not_ subsequently visited. This could imply you are prerendering too much, and using up valuable resources of the user for little benefit.
```

Comment:
```suggestion
In addition to measuring the impact of pages that are visited after a prerender, it is also important to measure pages that are prerendered and _not_ subsequently visited. This could imply you are prerendering too much, and using up valuable resources of the user for little benefit.
```

Not sure if this is a cultural difference, but I find it very confusing when sentences are started with "as well as..."